### ISSUE TITLE
Roadmap to release

### ISSUE THREAD

**Issue by helgee** [JuliaAstro/AstroBase.jl#45]
* [ ] Polish submodules
    * [x] `Bodies`
    * [X] `Constants`
    * [ ] `Coords`
        * [x] Figure out interpolation situation
    * [ ] `EarthAttitude`
    * [ ] `Ephemerides`
    * [ ] `Frames`
    * [ ] `Time`
    * [ ] `TwoBody`
    * [ ] `Util`
* [ ] Overhaul testing and improve coverage
* [ ] Tag release 🎉 

@giordano It seems that the LTS was bumped to 1.6. So the initial release should probably support 1.0. Don't you think?

**Comment by giordano** [comment]
> It seems that the LTS was bumped to 1.6. So the initial release should probably support 1.0. Don't you think?

Yeah, I saw that.  I'd say, let' try to support v1.0, but it's too much of a hassle move on.  Am I too much pragmatic? :laughing:

**Comment by helgee** [comment]
> Am I too much pragmatic? 😆

Nah, it's fine 😄

It should not be too much of problem. The issue right now is that I am using OrdinaryDiffEq.jl to generate test data, that package is 1.3+, and cannot find a version that does not clash with the other dependencies. But I should just hardcode that and get rid of the test-only dependency.

**Comment by helgee** [comment]
I have rolled my own cubic spline interpolation 😜
